packages/sdk/src/core/teamsfx.browser.ts

```diff
@@ -48,33 +48,33 @@ export class TeamsFx implements TeamsFxConfiguration {
   }
 
   private loadFromEnv(): void {
-    let env: any;
     if (window && (window as any).__env__) {
       // testing purpose
-      env = (window as any).__env__;
+      const env = (window as any).__env__;
+      this.configuration.set("authorityHost", env.REACT_APP_AUTHORITY_HOST);
+      this.configuration.set("tenantId", env.REACT_APP_TENANT_ID);
+      this.configuration.set("clientId", env.REACT_APP_CLIENT_ID);
+      this.configuration.set("initiateLoginEndpoint", env.REACT_APP_START_LOGIN_PAGE_URL);
+      this.configuration.set("applicationIdUri", env.M365_APPLICATION_ID_URI);
+      this.configuration.set("apiEndpoint", env.REACT_APP_FUNC_ENDPOINT);
+      this.configuration.set("apiName", env.REACT_APP_FUNC_NAME);
```

Comment:
Better extract these sentence to one function:

```
setConfigurationsFromEnv(env);
...
setConfigurationsFromEnv(process.env);
```

Reply:
Keeping `process.env.xxx` here to be replaced by webpack plugin.
https://webpack.js.org/plugins/environment-plugin/